app/main.py

```
@@ -2,0 +1,19 @@
+from app.cafe import Cafe
+
+
+from app.errors import VaccineError
+from app.errors import NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotWearingMaskError:
```

The order of exception handling should catch `VaccineError` before `NotWearingMaskError` to ensure that vaccine issues take precedence, as per the requirements: 'If a VaccineError is caught for any friend, immediately return "All friends should be vaccinated".' Currently, a mask error could be counted before a vaccine error is checked, which is incorrect.